docs/es/GIT.es.md

```diff
@@ -0,0 +1,43 @@
+# USO DE GIT:
+
+Hay algunas reglas que se aplican al interactuar con el repositorio del proyecto. Esto nos permitirá tener un estándar que cualquier desarrollador pueda entender. Es importante cumplir con estas reglas para que la herramienta de linting nos permita usar el repositorio.
+
+**Rama:**
+
+Cada vez que se le asigna un problema, debe crear una nueva sucursal. Esto se puede hacer a través del comando:
+
+```
+git branch <nombre de la rama>
+```
+
+Por conveniencia, las sucursales deben tener una estructura definida. Por ejemplo, si el issue se trata de agregar una nueva función, la rama debería ser:
```

Comment:
change "sucursal" for branch or rama please, only use a word to refer to the same thing since someone can confuse if read so terms